.github/scripts/therock_configure_ci.py

```diff
@@ -45,14 +96,21 @@ def retrieve_projects(args):
     if args.get("is_push"):
         subtrees = list(subtree_to_project_map.keys())
 
+    # If .github/*/therock* were changed, run all subtrees
+    base_ref = args.get("base_ref")
+    modified_paths = get_modified_paths(base_ref)
+    print("modified_paths (max 200):", modified_paths[:200])
```

Comment:
```suggestion
    print(f"modified_paths (max 200): {modified_paths[:200]}")
```

Prefer using f-strings, especially since f-strings are already used in print commands elsewhere in the python scripts.